Packs/Core/Playbooks/playbook-SSO_Password_Spray.yml

```diff
@@ -0,0 +1,1374 @@
+          "x": 1310,
+          "y": 380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 885d1ae0-3a02-46ee-891d-0177cdce2b17
+    type: regular
+    task:
+      id: 885d1ae0-3a02-46ee-891d-0177cdce2b17
+      version: -1
+      name: Check user risk level
+      description: Gets the risk score of the users who successfully logged in.
+      script: '|||core-list-risky-users'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "36"
+    scriptarguments:
+      user_id:
+        simple: ${SuccessfulLoginUsernames}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1340,
+          "y": 1035
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: d02e94a0-eda1-4729-804c-c17051c8c702
+    type: title
+    task:
+      id: d02e94a0-eda1-4729-804c-c17051c8c702
+      version: -1
+      name: Containment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1790,
+          "y": 1370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: b95fa692-f074-4cd7-868d-464005a3decb
+    type: condition
+    task:
+      id: b95fa692-f074-4cd7-868d-464005a3decb
+      version: -1
+      name: Were any successful login attempts?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "Yes":
+      - "28"
+      - "14"
+      - "39"
+    separatecontext: false
+    conditions:
+    - label: "Yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: SuccessfulLoginUsernames
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1100,
+          "y": 840
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 7da596f4-aa21-49e1-8640-366e9803b7cf
+    type: condition
+    task:
+      id: 7da596f4-aa21-49e1-8640-366e9803b7cf
+      version: -1
+      name: Are there any suspicious characteristics?
```

Comment:
change to:
Are there any suspicious users?

Reply:
Inside the condition I am checking if the suspicious users list is not empty, because I need users list to do all the actions later (clear session, password expire, etc, it based on users). But the actual thing that I want to check in this task is if any suspicious characteristics were found on those users. 
"I think Are there any suspicious characteristics?" is the correct one, but let me know if I should change it.
@umishkin @AdiPeret

Reply:
lets keep it characteristics, it's ok